### PR TITLE
Add entry - HTML: Tags: `<kbd>`

### LINKED ISSUE #569: [Term Entry] HTML: Tags: <kbd>
### Reason/inspiration (optional)

The `<kbd>` tag displays text as a key on a computer. For example:

<kbd>ctrl</kbd>+<kbd>c</kbd>

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Made the entry for the `<kbd>` tag.

Closes #569 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by ethan-master-coding** [comment]
Please link #569.

**Comment by Dusch4593** [comment]
> Please link #569.

Linked! Fwiw, when you first create the PR you can also write "Closes #[issue-number-here]" under the "Description" and it will set the issue to be closed after the connected PR is merged.

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>
+```
+
+Result:
+
+You can copy text by pressing ``CTRL`` + ``c``.
```
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/kbd/kbd.md:28]: This is a case where we could use an image of the rendered text. The picture can be placed in the /media folder.

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example 2
+
+```html
+<p>You can peaste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>v</kbd></kbd>.</p>
+```
+Result:
+
+You can peaste text by pressing ``CTRL`` + ``c``.
```
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/kbd/kbd.md:37]:
```suggestion
You can paste text by pressing ``CTRL`` + ``v``.
```

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The `<kbd>` tag makes characters look like they are keys on a keyboard.
+
+## Syntax
+
+All that is needed to be done is put the characters between an opening and closing tag. The `<kbd>` tag has no attributes. They can be nested but the result will be the same.
+
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>
```
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/kbd/kbd.md:23]: Applies to this and below. Generally, when referring to alpha keys on the keyboard, they're usually capitalized.

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<kbd>` tag makes characters look like they are keys on a keyboard.
+
+## Syntax
+
+All that is needed to be done is put the characters between an opening and closing tag. The `<kbd>` tag has no attributes. They can be nested but the result will be the same.
```
Comment (SSwiniarski) [inline content/html/concepts/tags/terms/kbd/kbd.md:18]: Should have a pseudo or html block with something like `<kbd></kbd>

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@ethan-master-coding, I did a first review and left a few comments.

File: content/html/concepts/tags/terms/kbd/kbd.md (hunk 4 above)
Comment (yangc95) [reply content/html/concepts/tags/terms/kbd/kbd.md:18]: `<kbd><!-- Text goes here --></kbd>`**

**Comment by ethan-master-coding** [comment]
@SSwiniarski All edits look good. I accepted the changes and made the changes requested.

**Review by SSwiniarski — APPROVED** [review]
Looks good. On to next review.

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
```
Comment (KTom101) [inline content/html/concepts/tags/terms/kbd/kbd.md:18]:
```suggestion
The `<kbd>` tag has no attributes. Text is placed in between the opening and closing `<kbd>` tags and will render in the browser's default monospace font. `<kbd>` can be nested but the result will be the same.
```

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+```pseudo
+<kbd><!-- Text goes here --></kbd>
+```
+  
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>
```
Comment (KTom101) [inline content/html/concepts/tags/terms/kbd/kbd.md:27]:
```suggestion
<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>C</kbd>.</p>
```

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+Result:
+
+![picture of the <kbd> in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb%20tag%20example%201.png)
+  
+## Example 2
+
+```html
+<p>You can peaste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>v</kbd></kbd>.</p>
```
Comment (KTom101) [inline content/html/concepts/tags/terms/kbd/kbd.md:37]:
```suggestion
<p>You can paste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>V</kbd></kbd>.</p>
```

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>c</kbd>.</p>
+```
+
+Result:
+
+![picture of the <kbd> in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb%20tag%20example%201.png)
```
Comment (KTom101) [inline content/html/concepts/tags/terms/kbd/kbd.md:32]: The file names for the images need to be changed to something like "kbd-tag-image.png" where  "-" is used for spacing. The image link will look something like this "https://raw.githubusercontent.com/Codecademy/docs/main/media/kbd-tag-image.png".  Take a look in the media folder to see how the names are written. Or you can take a look at the [recently merged  HTML `<span>` entry](https://github.com/Codecademy/docs/pull/605/files) which uses an image.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@ethan-master-coding Just finished taking a look at this. Here are some changes.

**Comment by ethan-master-coding** [comment]
@KTom101 All edits look good. I accepted the changes and made the change requested.

**Review by KTom101 — APPROVED** [review]
@ethan-master-coding  Thank you for making the changes.
Let's see if anyone wants to make any additional edits.

cc @yangc95 @Dusch4593

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>C</kbd>.</p>
+```
+
+Result:
+
+![picture of the <kbd> in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb-tag-example-1.png)
```
Comment (yangc95) [inline content/html/concepts/tags/terms/kbd/kbd.md:33]:
```suggestion
![Keyboard Tag Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-1.png)
```

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+  
+## Example 2
+
+```html
+<p>You can paste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>V</kbd></kbd>.</p>
+```
+  
+Result:
+
+![picture of the <kbd> tag in action](https://raw.githubusercontent.com/ethan-master-coding/docs/kdb-tag/media/kdb-tag-example-2.png)
```
Comment (yangc95) [inline content/html/concepts/tags/terms/kbd/kbd.md:43]:
```suggestion
![Keyboard Tag Example 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-2.png)
```

**Review by yangc95 — COMMENTED** [review]
@ethan-master-coding Took a quick look as well~

**Comment by ethan-master-coding** [comment]
@yangc95 All edits look good. I accepted the changes.

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+  
+## Example 2
+
+```html
+<p>You can paste text by pressing <kbd><kbd>CTRL</kbd>+<kbd>V</kbd></kbd>.</p>
+```
+  
+Result:
+
+![Keyboard Tag Example 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-2.png)
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/kbd/kbd.md:43]: We need to make a similar local change here by running the following command:

```
git mv kdb-tag-example-2.png kbd-tag-example-2.png
```

Then, run `git add` + `commit` + `push` back up to this branch.

File: content/html/concepts/tags/terms/kbd/kbd.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+  
+## Example 1
+
+```html
+<p>You can copy text by pressing <kbd>CTRL</kbd> + <kbd>C</kbd>.</p>
+```
+
+Result:
+
+![Keyboard Tag Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/kdb-tag-example-1.png)
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/kbd/kbd.md:33]: I noticed a small typo with the filenames. It should say "*kbd*-tag-example-1.png" (emphasis mine), instead. We will need to rename both this file and the other, too.

To make this change, you have to make it locally by running the following command:

```
git mv kdb-tag-example-1.png kbd-tag-example-1.png
```

Then, run `git add` + `commit` + `push` back up to this branch.

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Hey @ethan-master-coding just a few last suggested edits to make and then this PR will be set for merge! 👍

**Comment by ethan-master-coding** [comment]
@Dusch4593 I think I ran those commands correctly. Tell me if I'm wrong. (Never really used the terminal before only GH destop)

**Comment by Dusch4593** [comment]
> @Dusch4593 I think I ran those commands correctly. Tell me if I'm wrong. (Never really used the terminal before only GH destop)

Hmmm I don't the filenames are changed? That's ok, though! I'm happy to make that change if you'd like 😄

**Comment by ethan-master-coding** [comment]
@Dusch4593 You can go and make the change if you want.

**Comment by ethan-master-coding** [comment]
Also I it seems like the person reviewing #618 is inactive so the PR isn't going anywhere right now. What should we do about it?

**Comment by Dusch4593** [comment]
> Also I it seems like the person reviewing #618 is inactive so the PR isn't going anywhere right now. What should we do about it?

While you should definitely heed their feedback/suggestions, unfortunately their approval of the changes wouldn't counted as the one (1) needed for merge. 😦

It may be a little more time before either myself or one of my colleagues (@yangc95, @SSwiniarski, @KTom101, and @caupolicandiaz) make it to that one. I'll pop into it soon, though, to add to the thread concerning the GH checks.

**Comment by ethan-master-coding** [comment]
@Dusch4593 Got it. I should open an issue for the contribution guide for an edit for what each check does. And is @caupolicandiaz new? I haven't seen them in any PRs or Issues.

**Comment by Dusch4593** [comment]
> @Dusch4593 Got it. I should open an issue for the contribution guide for an edit for what each check does. And is @caupolicandiaz new? I haven't seen them in any PRs or Issues.

Nope, they're another one of us! 😄

**Comment by Dusch4593** [comment]
@ethan-master-coding I'm noticing on line # 36 that there are more `<kbd>` tags in that snippet than in the previous one (also no spacing around the "+"). Was this intentional or should I remove those extra tags?

**Comment by ethan-master-coding** [comment]
@Dusch4593 No the `+` isn't meant to be in the tags because it's not part of the hotkey. And I removed the extra spaces.

**Comment by Dusch4593** [comment]
@ethan-master-coding And what about what seem to be extra `<kbd>`s on line # 36?

**Comment by ethan-master-coding** [comment]
@Dusch4593 Its showing that they can be nested but it doesn't have an effect on the result

**Comment by Dusch4593** [comment]
> @Dusch4593 Its showing that they can be nested but it doesn't have an effect on the result

Doesn't that wrap the "+", though? If nesting has no effect on the rendered result, why don't we remove that outer pair of `<kbd>` tags? Less stuff to render/process.

**Comment by ethan-master-coding** [comment]
@Dusch4593 Well this is what MDN web docs day about the tag: 
```
Note: You don't need to do all this wrapping; you can choose to simplify it by leaving out the external <kbd> element. In other words, simplifying this to just <kbd>Ctrl</kbd>+<kbd>N</kbd> would be perfectly valid.

Note: Depending on your style sheet, though, you may find it useful to do this kind of nesting.
```

**Comment by Dusch4593** [comment]
I gotcha. So the nesting is really just a preference and has no meaningful effect on the final render. How about this, could we incorporate that information into both example sections?

* So maybe for "Example 1" precede the code snippet with that sentence that reads "Leaving out an external `<kbd>` wrapper element is value and will have no effect."
* Then, for "Example 2", precede that code snippet with something like "However, nested `<kbd>` tags may be useful for certain stylesheet setups."

**Comment by Dusch4593** [comment]
But hang on one quick second before you do that. Let me finish renaming those files so we have that piece done.

**Comment by ethan-master-coding** [comment]
@Dusch4593 Can't make the change now. I will do it in the morning

**Comment by ethan-master-coding** [comment]
@Dusch4593 Made the edit

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍

**Pull request merged**